docs/content/blog/protecting-applications-with-basic-auth/index.md

```
@@ -0,0 +1,212 @@
+
+# Common Use Cases
+
+## Monitoring and Health Checking
+Applications such as [Uptime Kuma] can make use of push-based health checking, ie an application sends an api request periodically with its current status. If Uptime Kuma is behind authelia (with authentication disabled), you can allow those push api requests using a service account. See [Path Bypass](#path-bypass) for how to bypass specific paths only.
+
+## Log Shipping
+Applications such as [Loki](https://grafana.com/docs/loki/latest/), [Mimir](https://grafana.com/docs/mimir/latest/), [Tempo](https://grafana.com/docs/tempo/latest/), and other log/metric/trace aggregators may not include their own authentication which makes this solution very useful. By configuring collection agents to use basic auth, they can ship logs or metrics to a protected application.
+
+# Advanced Configs
+
+### Path Bypass
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)